Library/Formula/openshift-cli.rb

```diff
@@ -0,0 +1,36 @@
+class OpenshiftCli < Formula
+  desc "OpenShift command-line interface tools"
+  homepage "https://www.openshift.com/"
+  url "https://github.com/openshift/origin.git",
+    :tag => "v1.1.1",
+    :revision => "e1d9873c1d5711b83fd3dd7eefe83a88ceb92c08"
+
+  head "https://github.com/openshift/origin.git"
+
+  depends_on "go" => :build
+
+  def install
+    # this is necessary to avoid having the version marked as dirty
+    ignore_file = buildpath/".git/info/exclude"
+    ignore_file.open("w") do |file|
+      file.puts "/.brew_home"
+    end
```

Comment:
@bfontaine Git puts a default file there, causing `#write` to fail with `Error: Will not overwrite /private/tmp/openshift-cli20160119-26167-1ubjvjy/.git/info/exclude`. I'll use `#atomic_write` instead.